test/unit/spec/common/jobManager-Spec.js

```diff
@@ -652,6 +719,48 @@ define([
                             expect(callArgs[0][0]).toEqual(`request-job-${actionString}`);
                             expect(callArgs[0][1].jobIdList.sort()).toEqual(expectedJobIds);
                         });
+
+                        it(`can ${action} a batch of jobs, including retries, in status ${status}`, async function () {
```

Comment:
Same thing -- cancelling / retrying a set of jobs, ensuring that if they are retries, the retry_parent is used.